play.py

```diff
@@ -0,0 +1,35 @@
+import torch
+from torch.distributions import Categorical
+import os
+from envs import GoHighEnv
+import atexit
+import platform
+
+checkpoint = torch.load(os.path.join("checkpoints", "agent.ckpt"))
+
+net = checkpoint["model"]
+
+options = dict(
```

Comment:
Minor thing: It would be better to have this as a configuration file that is read in so we can specify different configurations easily.